src/focus.service.ts

```diff
@@ -284,7 +284,14 @@ export class FocusService {
     this.focusRoot = newRootElem;
   }
 
-  public releaseFocus(scrollSpeed: number = Infinity) {
+  public releaseFocus(releaseElem?: HTMLElement, scrollSpeed: number = Infinity) {
+    if (releaseElem) {
+      if (releaseElem === this.focusRoot) {
+        this.releaseFocus(null, scrollSpeed);
+      }
+      return;
```

Comment:
Ah, I see. It might make sense to pop this out to a separate method like `releaseIfHeld`. Looks fine tho.